src/App.css

```diff
@@ -141,27 +139,34 @@ a:hover {
 }
 
 .coins-container {
-  flex-direction: column;
-  gap: 1rem;
+  width: 100%;
+  display: grid;
+  grid-template-columns: 50% 47%;
+  margin-bottom: 0.5rem;
   align-items: center;
   justify-content: center;
-  width: 100%;
-  padding-bottom: 1rem;
 }
 
 .card-container {
-  width: 74%;
+  width: 100%;
+  height: 7rem;
+  margin-top: 0.02rem;
+  margin-bottom: 0.1rem;
+  margin-right: 0.5rem;
 }
 
 .coin-card {
-  gap: 1rem;
-  justify-content: center;
+  column-gap: 0.02rem;
+  justify-content: left;
   align-items: center;
-  padding: 1rem;
-  background-color: rgb(46 47 53 / 80.1%);
+  padding-left: 0.5rem;
+  background-color: rgba(27, 28, 29, 0.801);
   backdrop-filter: blur(120px);
-  border-radius: 1rem;
+  border-radius: 0.3rem;
+  border: 0.1rem solid goldenrod;
   transition: ease-in-out 0.5s;
+  width: 100%;
```

Comment:
- [ ] you have a failing test, please look into that. Also, as mentioned by the previous reviewer, kindly test some of your functions including Redux and not only your components,
![Screenshot (994)](https://user-images.githubusercontent.com/65298048/234492684-07d4b8f6-178f-49f6-b0ae-16b543c367a3.png)